101-hdinsight-linux-with-edge-node/azuredeploy.json

```diff
@@ -0,0 +1,227 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterName": {
```

Comment:
is this globally unique (should note that in the description)?  better yet can we generate this w/ uniqueString()?